tests/Egret.Tests/Serialization/Audacity/AudacityImporterTests.cs

```diff
@@ -0,0 +1,193 @@
+
+            var tests = testsCases[0].Expect;
+
+            tests[0].Should().BeOfType<BoundedExpectation>();
+            tests[1].Should().BeOfType<BoundedExpectation>();
+            tests[2].Should().BeOfType<TemporalExpectation>();
+
+            var temporalTolerance = includeTests.TemporalTolerance ?? 0.5;
+            var spectralTolerance = includeTests.SpectralTolerance ?? 0.5;
+
+            tests.ForEach((e, index) =>
+            {
+                switch (index)
+                {
+                    case 0:
+                        e.Name.Should().Be($"Audacity Label test 1 in track Track 2 (0:0)");
+                        break;
+                    case 1:
+                        e.Name.Should().Be($"Audacity Label test 3 in track Track 2 (0:1)");
+                        break;
+                    case 2:
+                        e.Name.Should().Be($"Audacity Label test 2 in track Track 1 (1:0)");
+                        break;
+                }
+
+                switch (e)
+                {
+                    case BoundedExpectation bounded:
+                        switch (index)
+                        {
+                            case 0:
+                                bounded.Bounds.StartSeconds.Should().Be(4.0.WithTolerance(temporalTolerance));
+                                bounded.Bounds.EndSeconds.Should().Be(7.2446258503.WithTolerance(temporalTolerance));
+                                bounded.Bounds.HighHertz.Should().Be(10.0.WithTolerance(spectralTolerance));
+                                bounded.Bounds.LowHertz.Should().Be(1.0.WithTolerance(spectralTolerance));
+                                break;
+                            case 1:
+                                bounded.Bounds.StartSeconds.Should().Be(15.9714285714.WithTolerance(temporalTolerance));
+                                bounded.Bounds.EndSeconds.Should().Be(24.44.WithTolerance(temporalTolerance));
+                                bounded.Bounds.HighHertz.Should().Be(10000.0.WithTolerance(spectralTolerance));
+                                bounded.Bounds.LowHertz.Should().Be(10.0.WithTolerance(spectralTolerance));
+                                break;
+                            default:
+                                throw new XunitException();
+                        }
+                        break;
+                    case TemporalExpectation temporal:
+                        switch (index)
+                        {
+                            case 2:
+                                temporal.Time.StartSeconds.Should().Be(8.0228571429.WithTolerance(temporalTolerance));
+                                temporal.Time.EndSeconds.Should().Be(18.98.WithTolerance(temporalTolerance));
+                                break;
+                            default:
+                                throw new XunitException();
+                        }
+                        break;
+                }
+            });
```

Comment:
I would probably turn this into a static structure rather than use multiple switch statements